examples/example.lua

```diff
@@ -92,6 +92,39 @@ local decompress_deflate_with_level = LibDeflate:DecompressDeflate(
 	compress_deflate_with_level)
 assert(decompress_deflate_with_level == example_input)
 
+-- Compress with async coroutine, within World of Warcraft
```

Comment:
Your changes to examples/example.lua does not work outside World of Warcraft.
You need to ensure WoW specific code does not run out of WoW.
This is not a WoW specific library

Reply:
There is no need to use WoW API here.
Just use coroutine as an example